.gitlab-ci.yml

```diff
@@ -87,3 +88,21 @@ deploy_to_di_backend:manual:
     UPSTREAM_COMMIT_AUTHOR: $CI_COMMIT_AUTHOR
     UPSTREAM_TAG: $CI_COMMIT_TAG
     UPSTREAM_PACKAGE_JOB: build
+
+check_new_flaky_tests:
+  stage: quality-gate
+  extends: .testrunner
+  variables:
+      DD_SITE: datadoghq.com
+      DD_API_KEY: invalid_but_this_is_fine
+      DD_APP_KEY: invalid_but_fine
```

Comment:
Do you need these even if you do `export` below?

Reply:
Nope we don't, let me clean that up